src/components/Form/FormFields/TextFormField.tsx

```diff
@@ -24,6 +24,7 @@ export type TextFormFieldProps = FormFieldBaseProps<string> &
     trailingPadding?: string | undefined;
     leadingPadding?: string | undefined;
     suggestions?: string[];
+    isClear?: boolean;
```

Comment:
`isClear` doesn't make sense. One may not understand what this prop does by simply reading it.

```suggestion
    clearable?: boolean;
```